python/cudaq/__init__.py

```diff
@@ -41,6 +41,16 @@
 from .runtime.observe import observe
 from .mlir._mlir_libs._quakeDialects import cudaq_runtime
 
+try:
+    from qutip import Qobj, Bloch
+except ImportError:
+    raise Exception(
```

Comment:
maybe my Python is rusty but doesn't this exception here make the optional qutip dependency not optional... Everytime I `import cudaq` with this change I'll see an exception thrown. I don't think this is what we want.

Reply:
Hmm, that's true. Would it be better if the import silently fails if the dependency isn't installed? That way the user will simply get an error that the relevant function is not contained in the module.